packages/profile-sync-controller/src/sdk/authentication-jwt-bearer/types.ts

```diff
@@ -1,6 +1,6 @@
 import type { Env, Platform } from '../env';
 
-export const enum AuthType {
+export enum AuthType {
```

Comment:
Infura SDK ask. Projects that use `isolatedModules` are not compatible with const enums.